src/Views/batchprocessor.html

```diff
@@ -26,9 +26,20 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
         <!-- Start submit batch -->
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-           
+            <!-- Start instructions -->
+            <div id="submitBatchInstructions"> To submit a batch: <br>
+                <ul style="list-style-type: circle;">
+                    <li>All highlighted fields must be completed before submission</li>
+                    <li>Only points that are contained within the selected State/Region will be processed</li>
+                    <li>Batches are limited to 250 points each and if it contains more, only the first 250 points will be
+                    processed</li>
+                    <li>Only point shapefiles will be processed so no polygon, polyline, or multipoint geometries</li>
+                </ul>
+            </div>

```

Comment:
Suggested changes:
- Added additional context
- Changed the instructions to an ordered list so it reads as a sequential "to do" list

```suggestion
            <div> The StreamStats Batch Processor allows users to run StreamStats for numerous points via an automated process.
                <br>
                <br>
                <b>How to Submit a Batch</b>
                <br>
                <ol>
                    <li>Create a point shapefile:</li>
                    <ul style="list-style-type: disc; padding: 0px">
                        <li>Ensure points are coincident with the stream grid used by StreamStats for delineation.</li>
                        <li>Shapefile must have an ID field that uniquely identifies each point. </li>
                        <li>Shapefile must be a point shapefile. Polyline, polygon, and multipoint geometries cannot be processed.</li>
                        <li>Only the first 250 points will be processed.</li>
                        <li>Only points contained within the requested State/Region will be processed.</li>
                    </ul>
                    <li>Complete the form below. Highlighted fields are required.</li>
                    <li>Click the "Submit Batch" button and confirm that the batch was submitted successfully.</li>
                    <li>Check your email or the "Batch Status" tab to see the batch status and retrieve file geodatabase results.</li>
                </ol>
            </div>

```